vowpalwabbit/distributionally_robust.h

```diff
@@ -30,26 +43,8 @@ namespace distributionally_robust
     {
     }
     double qfunc(double w, double r) { return unbounded ? 1 : -(gamma + (beta + r) * w) / ((n + 1) * kappa); }
-
-    void save_load(io_buf& model_file, bool read, bool text)
-    {
-      if (model_file.num_files() == 0) { return; }
-      std::stringstream msg;
-      if (!read) { msg << "_duals_unbounded " << unbounded << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&unbounded), sizeof(unbounded), read, msg, text);
-
-      if (!read) { msg << "_duals_kappa " << kappa << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&kappa), sizeof(kappa), read, msg, text);
-
-      if (!read) { msg << "_duals_gamma " << gamma << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&gamma), sizeof(gamma), read, msg, text);
-
-      if (!read) { msg << "_duals_beta " << beta << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&beta), sizeof(beta), read, msg, text);
-
-      if (!read) { msg << "_duals_n " << n << "\n"; }
-      bin_text_read_write_fixed(model_file, reinterpret_cast<char*>(&n), sizeof(n), read, msg, text);
-    }
+    friend size_t VW::model_utils::process_model_field<>(
```

Comment:
If using `<>` then you should have `template <>`. In fact you should be able to remove `<>` if you add `template <>` as the template type can then be deduced from the args.

Additionally, why is this declaration different from that on line 26?

Reply:
This should be deleted. I was using a specialized template at first, but it works the same without any template.